docs/docs/lib/java.mdx

```diff
@@ -279,10 +279,22 @@ For more references, see the [Examples](#code-examples) below.
 
 #### Cacheing and refreshing behaviour
 
+As of version 0.9.0, there are 2 refresh strategies available.
+
+##### Stale While Revalidate
+
+This is the default strategy but can be explicitly stated by passing `FeatureRefreshStrategy.STALE_WHILE_REVALIDATE` as the refresh strategy option to the `GBFeaturesRepository` builder or constructor.
+
 The `GBFeaturesRepository` will automatically refresh the features when the features become stale. Features are considered stale every 60 seconds. This amount is configurable with the `ttlSeconds` option.
 
 When you fetch features and they are considered stale, the stale features are returned from the `getFeaturesJson()` method and a network call to refresh the features is enqueued asynchronously. When that request succeeds, the features are updated with the newest features, and the next call to `getFeaturesJson()` will return the refreshed features.
 
+##### Server-Sent Events
+
+This is a new strategy that can be enabled by passing `FeatureRefreshStrategy.SERVER_SENT_EVENTS` as the refresh strategy option to the `GBFeaturesRepository` builder or constructor.
```

Comment:
question: do we skip SWR when using this strategy? In the JS SDK, both happen at the same time.

Reply:
In the Java SDK we do. It seems redundant to do both—if you have an active SSE session then your features should already be up to date, so re-fetching them shouldn't cause a change if SSE is working correctly.

Reply:
fyi: we didn't do it this way for JS in case there were any connectivity issues on SSE and updates were missed.

Reply:
In the case of the Java SDK, a connectivity issue should result in the connection closing, which would trigger a reconnect. This section is in the Java docs you're looking at btw, not the JS or build-your-own docs.